subt/docker/build.bash

```diff
@@ -43,7 +43,8 @@ image_plus_tag=$image_name:$(date +%Y_%m_%d_%H%M)
 
 shift
 
-docker build --rm --no-cache -t $image_plus_tag --build-arg user_id=$user_id "$@" $DIR/$image_name
```

Comment:
I am not sure about the `--no-cache` ... it used to be "must have" to have fresh OSGAR build.

Reply:
That is no longer true. The Dockerfile contains COPY instruction that checks if any of the to-be-copied files has changed and if so, busts the cache. This didn't work for the "git clone" method.